### ISSUE TITLE
[Bug Report] Seemingly DNS failure after a short interval of time

### ISSUE THREAD

**Issue by LEXUGE** [daeuniverse/dae#604]
### Checks

- [X] I have searched the existing issues
- [X] I have read the documentation
- [X] Is it your first time sumbitting an issue

### Current Behavior

After a certain period of time, dae fails to dial with error:
```
dial to wss://SERVER: dial tcp: lookup SERVER: device or resource busy
```
However, the `dig SERVER` works just fine. Not sure if it's actually a DNS failure.

### Steps to Reproduce

My config:
```
global{
  wan_interface: auto

  log_level: info
  allow_insecure: false
  # auto_config_kernel_parameter: true
  dial_mode: domain
}

# Forward all DNS requests to local DNS
dns {
  upstream {
    dcompass: 'udp://127.0.0.1:53'
  }
  routing {
    request {
      fallback: dcompass
    }
  }
}

node {
 v2ray: "NODE DETAIL"
}

group {
  first {
    policy: fixed(0)
  }
}

routing{
  # Don't route dcompass's requests
  pname(NetworkManager, dcompass) -> must_direct

  domain(geosite:category-ads) -> block
  dip(geoip:private) -> direct
  dip(geoip:cn) -> direct
  # domain(geosite:cn) -> direct

  # Don't route games that have server in China.
  domain(geosite:category-games@cn) -> direct

  fallback: first
}
```

### Environment

- **Dae version (use `dae --version`)**: v0.7.0
- **OS (e.g `cat /etc/os-release`)**:
```
ANSI_COLOR="1;34"
BUG_REPORT_URL="https://github.com/NixOS/nixpkgs/issues"
BUILD_ID="24.11.20240804.cb9a96f"
DOCUMENTATION_URL="https://nixos.org/learn.html"
HOME_URL="https://nixos.org/"
ID=nixos
IMAGE_ID=""
IMAGE_VERSION=""
LOGO="nix-snowflake"
NAME=NixOS
PRETTY_NAME="NixOS 24.11 (Vicuna)"
SUPPORT_URL="https://nixos.org/community.html"
VERSION="24.11 (Vicuna)"
VERSION_CODENAME=vicuna
VERSION_ID="24.11"
```
- **Kernel (e.g. `uname -a`)**: `Linux x1c7 6.10.3 #1-NixOS SMP PREEMPT_DYNAMIC Sat Aug  3 07:01:09 UTC 2024 x86_64 GNU/Linux`
- **Others**:

**Comment by LEXUGE** [comment]
Seems like it's due to my node doesn't have IPv6 support while IPv6 was used. Disable IPv6 in dae `dns` resolves this issue. But I suspect dae should handle better?

**Comment by mzz2017** [comment]
的确，这种情况应该会被 connectivity check 检测到，你的 group 的 policy 是什么，如果是 fixed 可能会使得检测失效。
另外，你的 group 有几个节点？

**Comment by olicesx** [comment]
Closing as outdated: reported against a pre-v2.0 build/environment that has since been superseded by the v2.0.0 / #1099 datapath and control-plane rework, and no longer reproducible against current main. If this still reproduces on the latest release, please open a new issue with current-version logs. Housekeeping after #1099.

**Comment by olicesx** [comment]
Closing as outdated: reported against a pre-v2.0 build/environment that has since been superseded by the v2.0.0 / #1099 datapath and control-plane rework, and no longer reproducible against current main. If this still reproduces on the latest release, please open a new issue with current-version logs. Housekeeping after #1099.
